tools/buildsys/src/manifest.rs

```diff
@@ -33,6 +33,46 @@ url = "https://bar"
 sha512 = "123456"
 ```
 
+The `bundle-*` keys on `external-files` are a group of optional modifiers
+and are used to untar an upstream external file archive,
+vendor any dependent code,
+and produce an additional archive with those dependencies.
+Only `bundle-modules` is required when bundling an archive's dependences.
+
+`bundle-modules` is a list of module "paradigms" the external-file should
+be vendored through. For example, if a project contains a go.mod and go.sum file,
+"go" may be entered to vendor the dependencies.
+Currently, only "go" is supported.
+
+`bundle-path` is an optional argument that provides the filepath
+within the archive that contains the module.
+By default, the first top level directory in the archieve is used.
+So, for example, given a go project that has the necessary go.mod and go.sum
+files within the archive located at the filepath a/b/c,
+this `bundle-path` value should be `a/b/c`.
+Or, given a archieve with a single directory that contains a go project
+that has go.mod and go.sum files, this option may be omitted since the single
+top level directory will be used.
```

Comment:
nit: various formatting suggestions
```suggestion
So, for example, given a Go project that has the necessary `go.mod` and `go.sum`
files within the archive located at the filepath `a/b/c`,
this `bundle-path` value should be "a/b/c".
Or, given an archive with a single directory that contains a Go project
that has `go.mod` and `go.sum` files, this option may be omitted since the single
top-level directory will be used.
```